core/include/thread.h

```diff
@@ -278,11 +278,13 @@ struct _thread {
  */
 #define THREAD_PRIORITY_IDLE           (THREAD_PRIORITY_MIN)
 
+#ifndef THREAD_PRIORITY_MAIN
```

Comment:
would you mind moving this line below the doxygen block, like it is done for THREAD_STACKSIZE_MINIMUM?
Feel free to squash directly. ACK otherwise!